File: android/tutorials/tutorial2-composing-ribs-finish-code/src/main/java/com/uber/rib/root/logged_out/LoggedOutRouter.java
package com.uber.rib.root.logged_out;

import com.uber.rib.core.ViewRouter;

/**
 * Adds and removes children of {@link LoggedOutBuilder.LoggedOutScope}.
 */
public class LoggedOutRouter extends
        ViewRouter<LoggedOutView, LoggedOutInteractor, LoggedOutBuilder.Component> {

    public LoggedOutRouter(
            LoggedOutView view,
            LoggedOutInteractor interactor,
            LoggedOutBuilder.Component component) {
        super(view, interactor, component);
    }
}

File: android/demos/compose/src/main/kotlin/com/uber/rib/compose/root/main/logged_out/LoggedOutRouter.kt
package com.uber.rib.compose.root.main.logged_out

import com.uber.rib.core.BasicViewRouter

class LoggedOutRouter(
  view: LoggedOutView,
  interactor: LoggedOutInteractor
) : BasicViewRouter<LoggedOutView, LoggedOutInteractor>(view, interactor)
